src/__tests__/OrsDirections.cy.js

```diff
@@ -47,6 +106,54 @@ describe('Test Directions', () => {
         expect(json1['metadata']['query']['profile']).to.equal('cycling-regular')
         expect(json2['metadata']['query']['profile']).to.equal('foot-walking')
       })
+
+      it('sets customHeaders in request', (done) => {
+        orsDirections.calculate({
+          coordinates: [[8.690958, 49.404662], [8.687868, 49.390139]],
+          profile: 'driving-car',
+          customHeaders: {'Accept': 'application/json'}
+        }).then(function(json) {
+          expect(json['routes'].length).to.be.greaterThan(0)
+          expect(json['routes'][0].summary.distance).to.be.greaterThan(2300)
+          expect(json['routes'][0].summary.distance).to.be.lessThan(2500)
+          done()
+        })
+      })
+    })
+  })
+
+  context('unused functions', () => {
+    it('clear() clears everything but API key', () => {
+      orsDirections.defaultArgs = {
+        variable1: 'v1',
+        api_key: 'API key'
+      }
+      orsDirections.clear()
+      expect(orsDirections.defaultArgs).to.deep.equal({api_key: 'API key'})
+    })
+
+    it('clearPoints() clears only coordinates', () => {
+      orsDirections.defaultArgs = {
+        'coordinates': [[8.690958, 49.404662], [8.687868, 49.390139]],
+        'otherArg': [1, 2]
+      }
+      orsDirections.clearPoints()
+      expect(orsDirections.defaultArgs.coordinates.length).to.equal(0)
+      expect(orsDirections.defaultArgs.otherArg.length).to.equal(2)
+    })
+
+    context('addWaypoint', () => {
+      it('addWaypoint() adds coordinates', () => {
+        orsDirections.defaultArgs.coordinates = [[8.690958, 49.404662]]
+        orsDirections.addWaypoint([8.687868, 49.390139])
+        expect(orsDirections.defaultArgs.coordinates.length).to.equal(2)
+      })
+
+      it('addWaypoint() creates pÁroperty if not existing', () => {
+        orsDirections.defaultArgs = {}
+        orsDirections.addWaypoint()
+        expect(orsDirections.defaultArgs).to.have.property('coordinates')
+      })
     })
```

Comment:
probably split and squash to OrsDirections commit

Reply:
had left them in a separate commit, in case we did want to fix the problem with addWaypoint/addLocation, or at least so it's visible it is a bug we were aware of. However, if the functions are to be taken out of the code entirely, then there needs to be no trace of this test, of course